python/pyspark/pandas/frame.py

```diff
@@ -1310,6 +1310,138 @@ def corr(self, method: str = "pearson") -> "DataFrame":
         """
         return cast(DataFrame, ps.from_pandas(corr(self, method)))
 
+    def corrwith(
```

Comment:
Let's also list it up in the documentation (https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.pandas/frame.rst)

Reply:
Sure!